tests/Protocols/Fof.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * This file is part of GameQ.
+ *
+ * GameQ is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * GameQ is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace GameQ\Tests\Protocols;
+
+/**
+ * Test Class for Fistful of Frags
+ *
+ * @package GameQ\Tests\Protocols
+ */
+class Fof extends Base
+{
+    /**
+     * Test responses for Fistful of Frags
+     *
+     * @dataProvider loadData
+     *
+     * @param $responses
+     * @param $result
+     */
+    public function testResponses($responses, $result)
+    {
+        \GameQ\Tests\MockDNS::mockHosts([
```

Comment:
Please remove unneccesary code.
Only required if you use hostname in the query config